frontend/src/components/App/Seats/styles.ts

```diff
@@ -22,27 +22,24 @@ import { APP_SM } from '@/ui/Texts';
 import { media, space } from '@/ui/helpers';
 
 const SeatsStyled = styled.div`
-  align-items: center;
   display: flex;
   height: 100%;
-  justify-content: center;
+  width: 100%;
   overflow: hidden;
 
   .content {
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    gap: ${space(1)};
     align-items: center;
```

Comment:
I think it works better with flex, as it supports natively centered element. We don't need to add extra columns to center them. On the other hand, we can change the negative margin and weird width calculations using "gap", as it is supported in "flex"

Reply:
The thing with flex gap is that is less supported, for example with safari (we already had problems with that in Stooa). [Reference Can I Use](https://caniuse.com/?search=gap)

And the grid layout has the same center mechanisms as flex and you can target each element. I chose grid because the ui was literally a grid and we don't have to worry about child width and height, we just have to worry about the grid and 2 child position and also the support in gap that makes it cleaner.

Reply:
About the support, we don't have any specified support defined, but our designer had this problem while making reviews of other PR where I used flex-gap, so I supposed we shouldn't use it yet.

You are right about center mechanisms, I understood what you meant after writing the comment. Grid is less _flex_ible 🥷🏻